ksp_plugin_test/plugin_integration_test.cpp

```diff
@@ -353,16 +353,16 @@ TEST_F(PluginIntegrationTest, BarycentricRotatingNavigationIntegration) {
            -1.0 * AstronomicalUnit / Hour,
             0.0 * AstronomicalUnit / Hour}) * (t - initial_time);
     auto const& vessel = *plugin_->GetVessel(vessel_guid);
-    auto const history =vessel.history();
-    auto const psychohistory =vessel.psychohistory();
+  auto const history = vessel.history();
+  auto const psychohistory = vessel.psychohistory();
     auto const rendered_trajectory =
         plugin_->renderer().RenderBarycentricTrajectoryInWorld(
             plugin_->CurrentTime(),
             history->begin(),
             psychohistory->end(),
             sun_world_position,
             plugin_->PlanetariumRotation());
-    EXPECT_EQ(4321, rendered_trajectory.size());
+    EXPECT_THAT(rendered_trajectory.size(), AllOf(Ge(4321), Le(9414)));
```

Comment:
SizeIs(AllOf(Ge(4321), Le(9414))).

Reply:
Actually, SizeIs(AnyOf(4321, 9414)), it’s not like we should expect to fall somewhere in between with a three-quarters-debug configuration.

Reply:
Done (2x).